libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationTelemetry.java

```diff
@@ -177,24 +177,16 @@ void startSession(DirectionsRoute directionsRoute) {
   }
 
   void stopSession() {
-    if (navigationSessionState.startTimestamp() != null) {
-      NavigationMetricsWrapper.cancelEvent(
-        navigationSessionState, metricProgress, metricLocation.getLocation(), context
-      );
-    }
+    sendCancelEvent();
     gpsEventFactory.reset();
     resetDepartFactory();
   }
 
-  void endSession(boolean isConfigurationChange) {
-    this.isConfigurationChange = isConfigurationChange;
```

Comment:
@Guardiola31337 should we remove the `NavigationTelemetry` field `isConfigurationChange` as well?  And see if this has any adverse effects on the events sending.

Reply:
@danesfeder 
> should we remove the `NavigationTelemetry` field `isConfigurationChange` as well?

Sure thing! My bad 🤦‍♂️ 

![sonic](https://media.giphy.com/media/xT9IgtnH3FaseTdApq/giphy.gif)

>  And see if this has any adverse effects on the events sending.

>> We should test thoroughly that this change doesn't introduce any other issues.

Reply:
This is updated @danesfeder and ready for another round of 👀